packages/pds/src/services/repo/blobs.ts

```diff
@@ -124,6 +124,14 @@ export class RepoBlobs {
       .execute()
   }
 
+  async processRebaseBlobs(did: string, newRoot: CID) {
```

Comment:
Maybe a question for another day, but I think as-is blobs on deleted records will stick around, including after a rebase.  So there could be blobs associated with the rebase commit, but no record present in that commit referencing them.  Hmmm!

Reply:
ah that's a good point 🤔 

alright I guess I'll probably have to list repo-blobs then